### PR TITLE
Added like button functionality

### PR DESCRIPTION
**Pull request by tomaszVodafone**
@marcoarevalo @steerpike @zhyyin could you please review as it is my first attempt at meteor and there is a good chance I did something stupid that you boys might spot, cheers

### CODE REVIEW

File: give-praise/javascripts/client/PraiseBlogClient.js [hunk 1]
```diff
@@ -58,6 +58,12 @@ Template.myPosts.events({
 	}
 });
 
+Template.body.events({
```
Comment (thepig25) [inline give-praise/javascripts/client/PraiseBlogClient.js]: Cant have the call on the body. Need to move this to praiseBlog or praisePosts

File: give-praise/javascripts/models/PraiseBlogModel.js [hunk 2]
```diff
@@ -1,4 +1,11 @@
 /*global Mongo, PraisePosts */
 /*jshint -W020 */
 PraisePosts  = new Mongo.Collection('praisePosts');
+PraisePosts.allow({
```
Comment (thepig25) [inline give-praise/javascripts/models/PraiseBlogModel.js]: This should be in PraiseBlogServer.js. Only collection info should be in the model.

File: give-praise/templates/partials/praisePost.html [hunk 3]
```diff
@@ -19,6 +19,8 @@ <h2 class="praisePostValue">
 			<h2 class="praisePostUser">
 			Sent by {{username}}
 			</h2>
+			<button class="like btn btn-primary btn-lg" data-plaudit-id="{{_id}}">I Like!</button>
```
Comment (thepig25) [inline give-praise/templates/partials/praisePost.html]: feel like we should be able to get this info from the JS. Not as a data attribute.

File: give-praise/javascripts/client/PraiseBlogClient.js (hunk 1 above)
Comment (tomaszVodafone) [inline give-praise/javascripts/client/PraiseBlogClient.js]: Agree, but tried this didn't work, not sure why

File: give-praise/templates/partials/praisePost.html (hunk 3 above)
Comment (tomaszVodafone) [inline give-praise/templates/partials/praisePost.html]: agree, this._id didn't work, gave me undefined

**Pull request merged**